dashboard/test/ui/features/gdpr_dialog.feature

```diff
@@ -0,0 +1,36 @@
+@dashboard_db_access
+
+Feature: GDPR Dialog - data transfer agreement
+
+  Scenario: EU user sees the GDPR Dialog on dashboard, opt out
+    Given I am a teacher
+    Given I am on "http://studio.code.org/home?force_in_eu=1"
+    Then I wait to see ".ui-test-gdpr-dialog"
+    Then I wait to see ".ui-test-gdpr-dialog-logout"
+    Then I click selector ".ui-test-gdpr-dialog-logout"
+    Then check that I am on "http://code.org/"
+
+  Scenario: EU user sees the GDPR Dialog on dashboard, opt in
+    Given I am a teacher
+    Given I am on "http://studio.code.org/home?force_in_eu=1"
+    Then I wait to see ".ui-test-gdpr-dialog"
+    Then I click selector ".ui-test-gdpr-dialog-accept"
+    # Confirm dialog closed and I can do something on this page
+    And I create a new section
+
+  Scenario: GDPR Dialog privacy link works
+    Given I am a teacher
+    Given I am on "http://studio.code.org/home?force_in_eu=1"
+    Then I wait to see ".ui-test-gdpr-dialog"
+    Then I click selector ".ui-test-gdpr-dialog-privacy-link"
+    Then check that I am on "http://code.org/privacy"
+
+  Scenario: EU user sees the GDPR Dialog on dashboard, opt in, don't show again
+    Given I create a teacher named "Madame Maxime"
```

Comment:
You could also test using `I create a student in the eu named` and make sure that the dialog doesn't show for such a user who has already accepted the data transfer agreement during sign-up.